codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/customizations/InternalServerErrorDecorator.kt

```diff
@@ -14,20 +14,37 @@ import software.amazon.smithy.model.traits.ErrorTrait
 import software.amazon.smithy.model.traits.RequiredTrait
```

Comment:
Why not keeping both the decorators in the same file? They are similar for sure, but they can both be useful. WDYT?

Reply:
Sure. I'm just not sure how to name the file... 😵‍💫

Reply:
Ahahaha ok.. I would call the file AdditionalErrorsDecorator.kt..